src/types/references.types.ts

```diff
@@ -50,6 +51,12 @@ export interface Publisher {
   level: number | null;
 }
 
+export interface Chapter {
+  link: string;
+  pagesFrom: string;
+  pagesTo: string;
+  anthology: Book;
```

Comment:
noe i veien for å kalle denne book isteden for anthology?

Reply:
Tja. Er det ikke greit å spesifisere der man kan? Dette vil jo alltid være en bok av typen antologi.